webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DataSourceService.scala

```diff
@@ -251,6 +254,7 @@ class DataSourceService @Inject()(
           case layer: PrecomputedLayer => Some(layer.mags)
           case layer: ZarrLayer        => Some(layer.mags)
           case layer: Zarr3Layer       => Some(layer.mags)
+          case layer: WKWLayer         => Some(layer.mags)
```

Comment:
Maybe this match case statement is not needed anymore. :thinking: 
Needs investigation :mag: